src/styles/utils/_colors.scss

```diff
@@ -0,0 +1,7 @@
+$text-color: #060b35;
+$price-color: #616070;
+$buy-botton-color: #ffff;
```

Comment:
There seems to be a typo in the variable name `$buy-botton-color`. According to the [Code Style Rules](https://mate-academy.github.io/layout_task-guideline/html-css-code-style-rules), variable names should be spelled correctly. Consider renaming it to `$buy-button-color`.